config/splat.us.stno0.yaml

```diff
@@ -0,0 +1,65 @@
+options:
+  platform: psx
+  basename: stno0
+  base_path: ..
+  build_path: build/us
+  target_path: disks/us/ST/NO0/NO0.BIN
+  asm_path: asm/us/st/no0
+  asset_path: assets/st/no0
+  src_path: src/st/no0
+  ld_script_path: build/us/stno0.ld
+  compiler: GCC
+  symbol_addrs_path:
+    - config/symbols.us.txt
+    - config/symbols.us.stno0.txt
+  undefined_funcs_auto_path: config/undefined_funcs_auto.us.stno0.txt
+  undefined_syms_auto_path: config/undefined_syms_auto.us.stno0.txt
+  find_file_boundaries: true
+  use_legacy_include_asm: false
+  migrate_rodata_to_functions: true
+  asm_jtbl_label_macro: jlabel
+  symbol_name_format: us_$VRAM
+  section_order:
```

Comment:
could we add `disassemble_all: true` for this work? https://github.com/Xeeynamo/sotn-decomp/pull/1689

Reply:
Done, thanks for catching this